conf/systemd/sync_images_s3@.service

```
@@ -0,0 +1,13 @@
+# service instance name "%i" is off only (for now)
+[Unit]
+Description=Synchronize images to AWS S3 %i
+# __ will be replaced by @ in email-failures@.service
+OnFailure=email-failures@sync_images_s3__%i.service
+
+[Service]
+Type=oneshot
+User=off
+Group=off
+WorkingDirectory=/srv/%i/scripts/sync-s3-images
+ExecStart=.venv/bin/python3 sync_s3_images.py /mnt/off/images/products
```

Maybe we don't need to use a "@" here as you don't seem to really use the %i for now :-)

Or either we use it here:
```suggestion
ExecStart=.venv/bin/python3 sync_s3_images.py /mnt/%i/images/products
```